### PR TITLE
Adding create method for download station

### PR DESCRIPTION
**Pull request by VincentRoma**

### CODE REVIEW

File: synology_api/downloadstation.py [hunk 1]
```diff
@@ -105,6 +105,14 @@ def tasks_info(self, task_id, additional_param=None):
 
         return self.request_data(api_name, api_path, req_param)
 
+    def create_task(self, uri):
```
Comment (N4S4) [inline synology_api/downloadstation.py:108]: As there are many method would you add create_uri_task to be more specific on what the function does

**Review by N4S4 — CHANGES_REQUESTED** [review]
Hey Vincent thank you for the support!!
Would uou modify according to the comment? 
Just to be more slecific on what the function does

File: synology_api/downloadstation.py (hunk 1 above)
Comment (VincentRoma) [reply synology_api/downloadstation.py:108]: Sure, but I just named it after the official API doc: https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/DownloadStation/All/enu/Synology_Download_Station_Web_API.pdf#%5B%7B%22num%22%3A56%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C466%2C376%2C0%5D

The param can be either HTTP/FTP/magnet/ED2K links or the file path, so I can maybe name it create_download_task and name the param "link"

Comment (N4S4) [reply synology_api/downloadstation.py]: Ok so its tested that works with files? Because u did not test it yet as in not home

**Review by N4S4 — APPROVED** [review]

**Pull request merged**